src/Command/Command.php

```diff
@@ -471,28 +471,20 @@ public function queryAll(): array
         return [];
     }
 
-    public function queryColumn(): array|false
+    public function queryColumn(): array
```

Comment:
`![] === true` 

This code will be work fine. But it can be simplified and immediately return the result (`return $this->db->createCommand(...)->queryColumn();`)

Reply:
Add adopt PRs (see issue description)